auth-web/src/components/auth/InviteUsersForm.vue

```diff
@@ -135,8 +138,24 @@ export default class InviteUsersForm extends Vue {
     }
   }
 
+  private hasDuplicates (): boolean {
```

Comment:
i am not very sure  , so just wondering if this be achieved using  new Set(array)).size !== array.length? If not , its good.

Reply:
Great idea - done :)